src/tests/conformance/post_handshake/query/ping.rs

```diff
@@ -0,0 +1,50 @@
+use tempfile::TempDir;
+
+use crate::{
+    protocol::codecs::payload::{Payload, PingData},
+    setup::node::Node,
+    tools::synthetic_node::SyntheticNodeBuilder,
+};
+
+#[tokio::test]
+#[allow(non_snake_case)]
+async fn c009_PING_PING_REPLY_send_req_expect_reply() {
```

Comment:
Isn't this time to introduce some convenience function for simple request -> response tests? Could be done in another PR as this would be refactoring.

Reply:
yeah, maybe later. At the moment I really like this simplicity & readability over code repetition.
Like if someone (who has no idea how Rust works) opens our test case, it will find everything in it. That's my current feeling at least 😅